### ISSUE TITLE
I want to get labels of every cluster about optimalK

### ISSUE THREAD

**Issue by Li-Xue** [milesgranger/gap_statistic#3]
By your package, I got result of Optimal clusters，Thanks. I want to get labels of every cluster about optimalK. But I don't know how to change the code? Can you give me some help?Thanks.

**Comment by milesgranger** [comment]
Hello @Li-Xue , first thank you for using my project! Unfortunately,  what you are trying to do is outside the scope of this project. The purpose of this is to determine the _number_ of clusters. You then use that number to train a model, such as kmeans to that _number_ of clusters. As I understand it, the solution is fairly straight forward, and even demonstrated in the notebook found in this project. You obtain the number fom optimalK and then use that to train KMeans from sklearn. From there you will get the cluster labels. :)

**Comment by Li-Xue** [comment]
Thanks. I got it.
